### PR TITLE
Project 3 of the HTML/CSS Course: New York Times Clone

### PR DESCRIPTION
**Pull request by VanessaAoki**
We made a clone of the new york times article's page, it has diffent sections of the page, where we used float, flexbox and grid to style it. The page works best on a 1060x800 resolution and using the firefox or google chrome browser, assets might look out of place using other browsers or resolutions.

The focus of this branch was to make everything fit in the right places, setting up a good looking website.

### CODE REVIEW

File: assets/styles.css [hunk 1]
```diff
@@ -0,0 +1,842 @@
...
+  width: 100%;
+}
+
+main figcaption {
+  color: #666;
+  font-size: 0.9375rem;
+  width: 100%;
+  margin-left: auto;
+  margin-right: auto;
+}
```
Comment (jubaan) [inline assets/styles.css]: - [x] Change these inefficient selectors. and others found in the `style.css` file for better and more specific class selectors.

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## :wave: :wave: Hello @VanessaAoki, @kamaukk , and @kimothokamau  . Thanks for submitting you project!

Great thing this project has and has done well:
:100: has a navbar, the main content, and a footer
:100: Uses semantic HTML

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [x] The NYT logo should be a link, both in `<navbar>` and in `<footer>`

- [x] The images, titles of articles, titles, sections should be links with `<a>` anchor tag.
image
image
image
image

- [x] Don't use inefficient selectors in the CSS. See inline comment.

- [x] This title is out of place
image

### [OPTIONAL]  :yellow_circle:  

- [ ] All `` should have and `alt` attribute with a short and meaningful description.

- [ ] Change the name of the branch to something descriptive and meaningful related to the project.  Follow this tutorial to do it [LINK](https://www.w3docs.com/snippets/git/how-to-rename-git-local-and-remote-branches.html)

- [x]  Remove `cursor: pointer` property from css and change all elements that use it to `<a>` tags with `href` attribute.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Comment by kimothokamau** [comment]
Hello, thanks for the feedback. We have worked on the mandatory and optional recommendations that had been pointed out. 
Can you please review our code again?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,631 @@
...
+        <link href="https://fonts.googleapis.com/css2?family=PT+Serif:ital,wght@1,700&display=swap" rel="stylesheet">
+        <script src="https://use.fontawesome.com/e18ff5de8c.js"></script>
+        <script src="https://kit.fontawesome.com/0348c0adc9.js" crossorigin="anonymous"></script>
+    </head>
+
+    <body>
+        <nav>
+            <div class="top-left">
+                <i class="fa fa-bars" aria-hidden="true"></i>
+                <i class="fa fa-search" aria-hidden="true"></i>
```
Comment (descholar-ceo) [inline index.html]: These two elements should be clickable

File: index.html [hunk 3]
```diff
@@ -0,0 +1,631 @@
...
+                <a href="">SPACE & COSMOS</a>
+            </div>
+            <a href="#">
+                <img src="./assets/imgs/nyt-logo.png" alt="New York Times"></a>
+            <div class="top-right-btns">
+                <button>SUBSCRIBE NOW</button>
+                <button>LOG IN</button>
```
Comment (descholar-ceo) [inline index.html]: Give these two buttons `type` attribute. Example: `type="button"`

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @VanessaAoki and @kamaukk   :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [x] Hamburger and search buttons in navbar leftmost side should be clickabel
- [x] All buttons should have type attributes, with different values such as `button`
- [x] You should have a fixed footer on the bootom, check the original site
- [x] Please add some spacing between each box in `Suggested newsletters for you` and refactor it to make it look like the original one, currntly it's overriding the most popular articles

| Your clone | Original page |
| --- | --- |
|Screenshot from 2020-11-04 00-10-47|Screenshot from 2020-11-04 00-09-38|
:point_right: Could you see the difference :question: :question: :question: 

- [x] Implement all changes I suggested in inline comments in your codes


If anything seems to be unclear to you, don't hesitate to contact me for more clarification slack: `@Mugirase Emmanuel` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Comment by kimothokamau** [comment]
Hello, thanks for the feedback. We have fixed the recommendations as advised. Kindly review our code again.

**Review by descholar-ceo — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
## :clap: :clap: :clap: Congratulations @VanessaAoki and @kamaukk   , your project is approved :tada: :tada: :tada: 
You have implemented all minimum requirements for this project to mark it as complete, it doesn't need more changes, you can go ahead and merge all of the changes on base branch,

Don't forget to fill the mandatory project completion form, and see you to next project,

Happy coding :computer: :computer: :computer: 

Re-reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Comment by kimothokamau** [comment]
Thanks alot for the positive feedback.

**Pull request merged**